acme-numbers-events.js

```diff
@@ -1,16 +1,88 @@
-console.log('hi');
+class Game {
+  constructor() {
+    this.gameNumbers = [];
```

Comment:
what is the purpose of `this.gameNumbers = [];`? i see now that it's being used to store the amount of numbers, but the same can be achieved (and even better) by just using a variable
```suggestion
    this.amountOfNumbers = 0;
```